### DISCUSSION TITLE
New theme idea. Look it. 👀

### DISCUSSION THREAD

**Discussion by volkantash** [community/community#9680] [Other Feature Feedback, Questions, & Ideas]
A useful placement. Do you like it?
![resim](https://user-images.githubusercontent.com/25375593/148303624-af5da505-fcef-4652-a2c1-7986d97f67b6.png)

**Comment by gaknoll** [comment]
Thanks for sharing this concept @volkantash! We are currently working on updated global navigation. Aside from repos and access to GitHub's features, are there any other persistent links or shortcuts you would like to have in a potential sidebar?

**Reply by volkantash** [reply]
İlginiz için sağ olunuz. Örgütlenimi (organizasyonu) bu biçimde yönetmek işleri hızlandırabilir. Takım tartışmaları ve başka özellikleri de düşününce bu döşenek (layout) düzeni yakşı kalacaktır. Githubda takım arkadaşlarımız arkadaş dizmeme eklemek isterim. Ve sol yanda, arkadaşlarım ile ilgili kullanışlı özellikler kötü olmaz idi. Bir arkadaşımın adını pinlediğimde yaptığımda karşıma çıkan içerikler ve dahası onlarla ilgili çıkabilir. Takım olarak çalıştığımız sırada takımı seçmek güzel olabilir. Repolara zaten dizme (liste) oluşturabiliyoruz. Yine bu özellik sol yanda kullanılabilir. Bildirimleri yine sol yana taşıyabilir ve repo repo, takım takım, kişi kişi karşımıza getirebilirsiniz. Bugünlük diyebileceklerim bu kadar. :)

Thank you for your interest. Managing the organization (organization) in this way can speed things up. Considering the team discussions and other features, this layout will be appropriate. I would like to add our teammates to my friends list on Github. And on the left side, useful features related to my friends would not be bad. When I pin a friend's name, the content that I come across when I do and more may be related to them. It can be nice to choose the team while we work as a team. We can already create a list (list) for the repos. Again, this feature can be used on the left side. You can move the notifications to the left side and bring them repo repo, team by team, person by person. That's all I can say for today. :)

**Reply by volkantash** [reply]
Selam @gaknoll 👋 İşlerinizin ne durumda olduğunu merak ettim. Size daha çok katkıyı nereden ve nasıl sağlayabilirim? 

Hi @gaknoll 👋 I was wondering how your business is doing. Where and how can I contribute more to you?
